fortran/simulated-annealing/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           fortran 1.0
+PortGroup           linear_algebra 1.0
+
+github.setup        jacobwilliams simulated-annealing 57e249338abb97cb23a51018ea9152f5dc5a6c43
```

Comment:
can upstream be convinced to use tags and/or actually release versions...?

Reply:
To be honest, I would rather not trouble upstream unnecessarily, since this developer has numerous Fortran ports, many of which we use, so I will have more credible reasons to ask for help time to time.

Reply:
I know and it's annoying for all those ports that there are no tags and/or releases; so I would think that's just a better reason to ask/educate upstream on how to help downstream packaging.

Reply:
Let me just try building from the old tag; since we do not build documentation with `ford`, fixes since then can be ignored.

Reply:
@reneeotten Done, this is fine, since no FPM-related changes since then. (That is a concern, since FPM randomly introduces breaking changes on an ongoing basis.)